File: HelloWorld_CPP/LibHello/source/PrintHello.cpp
#include <PrintHello.h>

using namespace std;

PrintHello::PrintHello(string who, int times) : who( who ), times( times ) {}
		
void PrintHello::PrintNow(){
	for(int i = 0 ; i < this->times ; i++ ){
		cout << " Hello, " << who << "!" << endl;  
	}
}

void PrintHello::PrintNow(int times){
	this->times = times;
	this->PrintNow();
}
	

File: HelloWorld_PY/PrintHello.py
class Printer():
    def __init__( self, who, times = 1 ):
        self._who = who
        self._times = times
         
    def PrintHello( self ):
        for i in range( int( self._times ) ):
            print (" Hello, ", self._who, " !!!")
        print("\n")